lib/models/manifest.rb

```diff
@@ -1,3 +1,56 @@
+require_relative '../query_dsl'
+
 class Manifest < ActiveRecord::Base
+  include Elasticsearch::Model
+
+  ActiveRecord::Base.raise_in_transactional_callbacks = true
+
+  after_commit on: [:create] do
```

Comment:
using callbacks is the preferred way for indexing. See https://github.com/elastic/elasticsearch-rails/tree/master/elasticsearch-model#user-content-asynchronous-callbacks